src/fides/config/utils.py

```diff
@@ -66,5 +66,5 @@ def get_dev_mode() -> bool:
         "active_default_storage_type",
     ],
     "consent": ["override_vendor_purposes"],
-    "admin-ui": ["enabled", "url"],
+    "admin_ui": ["enabled", "url"],
```

Comment:
@galvana I think we meant to expose this here, but double checking that it is safe to show

@daveqnet this is adding admin ui settings to the `CONFIG_KEY_ALLOWLIST`

Reply:
Yes! This is safe, it's just the Admin UI URL

Reply:
thanks for confirming 👍